main.py

```diff
@@ -264,6 +263,24 @@ def buildGRMsg():
             await client.delete_message(new_gamerole_msg)
 
 
+async def protected_game_channels():
+    await client.wait_until_ready()
+    members = client.get_all_members()
+    server = client.get_server(test_server if test else nues_server)
+    with open('game_roles.json') as f:
+        game_roles = json.load(f)
```

Comment:
I don't understand why this is being loaded here. This could cause concurrency issues down the road.

Reply:
The game roles? I did that to check the game roles from our file